packages/vscode-extension/src/debug/teamsfxTaskHandler.ts

```diff
@@ -75,11 +74,16 @@ function isTeamsfxTask(task: vscode.Task): boolean {
       return (
         command !== undefined &&
         (command.trim().toLocaleLowerCase().endsWith("start") ||
-          command.trim().toLocaleLowerCase().endsWith("watch") ||
-          command.trim().toLowerCase() === "dev" ||
-          command.trim().toLowerCase() === "watch")
+          command.trim().toLocaleLowerCase().endsWith("watch"))
       );
     }
+
+    // dev:teamsfx and watch:teamsfx
+    const commandLine: string | undefined =
+      task.execution && (<vscode.ShellExecution>task.execution).commandLine;
+    if (commandLine !== undefined) {
```

Comment:
Is it possible that user uses command and args instead of full command line?

Reply:
Good catch. Yes it's possible, added `command` and `args` support.